File: JS/Arrays/WaveArray.js
module.exports = {
	//param A : array of integers
	//return a array of integers
	wave : function(A){
        A.sort(function(a, b){return a- b;});
        for(var i = 1; i < A.length; i+= 2){
            var v = A[i];
            A[i] = A[i - 1];
            A[i - 1] = v;
        }
        return A;
	}
};

File: Java/Arrays/WaveArray.java
public class WaveArray {
	public ArrayList<Integer> wave(ArrayList<Integer> a) {
	    Collections.sort(a);
	    for(int i = 0; i < a.size(); i += 2){
	        int ai = a.get(i);
	        if(i + 1 >= a.size())break;
	        a.set(i, a.get(i+1));
	        a.set(i+1, ai);
	    }
	    return a;
	}
}
